src/seals/diagnostics/__init__.py

```diff
@@ -67,3 +67,42 @@
     entry_point="seals.diagnostics.sort:SortEnv",
     max_episode_steps=6,
 )
+
+
+def register_cliff_world(suffix, kwargs):
+    """Register a CliffWorld with the given suffix and keyword arguments."""
+    gym.register(
+        f"seals/CliffWorld{suffix}-v0",
+        entry_point="seals.diagnostics.cliff_world:CliffWorldEnv",
+        kwargs=kwargs,
+    )
+
+
+for width, height, horizon in [(7, 4, 9), (15, 6, 18), (100, 20, 110)]:
```

Comment:
This is nitpicky, but all these global variables `width`, `height`,` `horizon` etc will now be polluting the module namespace. That is, `seals.diagnostics.width` will end up getting set to `100`.

I suggest putting all this in a function e.g. `def register_all_cliff_worlds` and calling that to convert the globals into local variables.